tests/unit/backend/recipe/algorithm/test_FitMultiplePeaksAlgorithm.py

```diff
@@ -0,0 +1,51 @@
+import os
+import unittest.mock as mock
+
+with mock.patch.dict(
+    "sys.modules",
+    {
+        "snapred.backend.log": mock.Mock(),
+        "snapred.backend.log.logger": mock.Mock(),
+    },
+):
+    from mantid.simpleapi import LoadNexusProcessed
+    from snapred.backend.dao.calibration.Calibration import Calibration
+    from snapred.backend.dao.CrystallographicInfo import CrystallographicInfo
+    from snapred.backend.dao.FitMultiplePeaksIngredients import FitMultiplePeaksIngredients
+    from snapred.backend.recipe.algorithm.FitMultiplePeaksAlgorithm import (
+        FitMultiplePeaksAlgorithm,  # noqa: E402
+    )
+    from snapred.meta.Config import Resource
+
+    def test_init():
+        """Test ability to initialize purge overlapping peaks algo"""
```

Comment:
update comments

Reply:
Thank you, comments have been updated to reflect the correct algorithm name